### ISSUE TITLE
Horizontal Page Links broken after Spicetify v2.35.

### ISSUE THREAD

**Issue by invisiblefmx** [Comfy-Themes/Spicetify#184]
**Environment**
- Spotify Version: `1.2.33.1042.g26c92729`
- Spicetify Version: `v2.35.1`
- Platform: `Win 11 (64bit)`
- Runtime: `Chrome`

**Describe the bug**

The "Horizontal Page Links" option is broken. It only shows the first two horizontal and the rest of them vertical after Spicetify v2.35 release.

**Screenshots**

Ignore the different symbol of the second custom app. I just only had an old screenshot with other customapps installed.

![1](https://github.com/Comfy-Themes/Spicetify/assets/157469657/80ad89a8-52c6-40cd-a47c-fa9a0bd838de)
after v2.35

![2](https://github.com/Comfy-Themes/Spicetify/assets/157469657/74ac7861-6e73-4854-9461-a26987c7df99)
before v2.35

**Comment by ohitstom** [comment]
will be resolved once beta branch is merged into main
